inc/vcf/error.hpp

```diff
@@ -233,11 +232,14 @@ namespace ebi
     #pragma db object
     struct IdBodyError : public BodySectionError
     {
-        using BodySectionError::BodySectionError;
-        IdBodyError() : IdBodyError{0} {}
-        IdBodyError(size_t line) : IdBodyError{line, "ID is not a single dot or a list of strings without semicolons or whitespaces"} { }
+        IdBodyError(size_t line = 0,
+                    const std::string &message = "ID is not a single dot or a list of strings without semicolons or whitespaces",
+                    ErrorFix error_fix = ErrorFix::IRRECOVERABLE_VALUE)
+                : BodySectionError{line, message}, error_fix{error_fix} { }
         virtual ~IdBodyError() override { }
         virtual void apply_visitor(ErrorVisitor &visitor) override { visitor.visit(*this); }
+
+        ErrorFix error_fix;
```

Comment:
Yes, but as it was needed in only 4 classes and not all, I didn't extract it😅. Another parameter called `field` is also common to many classes but not derived from the parent class. So in case `error_fix` should be derived from `class Error`, how about deriving `field` too?